helix-core/src/selection.rs

```diff
@@ -102,6 +109,20 @@ impl Range {
         self.anchor == self.head
     }
 
+    /// `true` when head > anchor.
+    #[inline]
+    #[must_use]
+    pub fn is_forward(&self) -> bool {
+        self.head > self.anchor
+    }
+
+    /// `true` when head < anchor.
+    #[inline]
+    #[must_use]
+    pub fn is_backward(&self) -> bool {
+        self.head < self.anchor
+    }
+
```

Comment:
Let's return a `Direction::Forward`/`Backward`, there's already an enum used by `movement.rs` I think? It's exported from `helix-core`

Reply:
Oh nice, I was contemplating making a new type when I wrote this, but I'm glad to learn this already exists. This is done.